rocAL/rocAL/include/augmentations/geometry_augmentations/node_flip.h

```diff
@@ -25,23 +25,22 @@ THE SOFTWARE.
 #include "parameter_vx.h"
 #include "parameter_factory.h"
 
-class FlipNode : public Node
-{
+class FlipNode : public Node {
 public:
     FlipNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     FlipNode() = delete;
-    void init(int flip_axis);
-    void init(IntParam *flip_axis);
+    void init(int h_flag, int v_flag);
+    void init(IntParam *h_flag_param, IntParam *v_flag_param);
     unsigned int get_dst_width() { return _outputs[0]->info().max_shape()[0]; }
     unsigned int get_dst_height() { return _outputs[0]->info().max_shape()[1]; }
     vx_array get_src_width() { return _src_roi_width; }
     vx_array get_src_height() { return _src_roi_height; }
-    vx_array get_flip_axis() { return _flip_axis.default_array(); }
+    vx_array get_flip_axis() { return _horizontal.default_array(); }
```

Comment:
Why are we returning only horizontal axis array?
Maybe we need 2 separate api's now to return for both x, y direction

Reply:
Will change in next PR